index.html

```diff
@@ -5,12 +5,53 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
+    <link rel="stylesheet" href="style.css">
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
 </head>
 
 <body>
-    <!-- Add your markup here -->
+    <header>
+      <img id="header-image" src="images/cupcake.png" alt="">
+      <nav id="nav-mobile">≡</nav>
+      <h2 id="header-title">The best cakes in town delivered to your door</h2>
+    </header>
+    <nav id="nav-desktop">
+      <ul id="nav-desktop-list">
+        <li class="nav-desktop-list-item">Home</li>
+        <li class="nav-desktop-list-item">Cakes</li>
+        <li class="nav-desktop-list-item">Order In</li>
+        <li class="nav-desktop-list-item">Lessons</li>
+        <li class="nav-desktop-list-item">About</li>
+      </ul>
+    </nav>
+    <main>
+      <section id="section-one">
+        <h1 id="section-one-title">Welcome</h1>
+        <p id="section-one-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna
+        aliqua. Eleifend quam adipiscing vitae proin sagittis nisl rhoncus mattis rhoncus. Fermentum iaculis eu non diam
+        phasellus vestibulum lorem sed. Fames ac turpis egestas sed. Nullam non nisi est sit amet. Id diam vel quam elementum
+        pulvinar etiam. Ullamcorper malesuada proin libero nunc consequat. Consequat mauris nunc congue nisi. Arcu vitae
+        elementum curabitur vitae.</p>
+      </section>
+      <section id="section-two">
+        <img id="section-two-image" src="images/cake.png" alt="">
+      </section>
+      <section id="section-three">
```

Comment:
This section doesnt have a heading or any text content so a div would be good here